### DISCUSSION TITLE
Is it possible to call multiple commands on a single key binding?

### DISCUSSION THREAD

**Discussion by AndrewKovalenko** [zed-industries/zed#8062] [Feature Requests]
Does Zed's config system support calling multiple commands on for the same key binding?
I'm trying to achieve something like:
```json
    "context": "Editor && vim_mode == insert && !menu",
    "bindings": {
      "j j": [
        "workspace::Save",
        "vim::NormalBefore"
      ]
```

**Comment by Moshyfawn** [comment]
Check out #7033

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
